interfaces/cython/cantera/kinetics.pyx

```diff
@@ -452,6 +456,281 @@ cdef class Kinetics(_SolutionBase):
         def __get__(self):
             return get_species_array(self, kin_getNetProductionRates)
 
+    property jacobian_settings:
+        """
+        Property setting behavior of Jacobian evaluation.
+
+        For ``GasKinetics``, the following keyword/value pairs are supported:
+
+        -  ``skip-third-bodies`` (boolean) ... if `False` (default), third body
+           concentrations are considered for the evaluation of jacobians
+
+        -  ``skip-falloff`` (boolean) ... if `True` (default), third-body effects
+           on reaction rates are not considered.
+
+        -  ``rtol-delta`` (double) ... relative tolerance used to perturb properties
+           when calculating numerical derivatives.
+
+        The Jacobian settings are updated using a dictionary::
+
+            >>> gas.jacobian_settings = {"skip-falloff": True}
+
+        which updates settings used for the evaluation of Jacobians. Passing an empty
+        dictionary will reset all values to their defaults.
+        """
+        def __get__(self):
+            cdef CxxAnyMap settings
+            self.kinetics.getJacobianSettings(settings)
+            return anymap_to_dict(settings)
+        def __set__(self, settings):
+            self.kinetics.setJacobianSettings(dict_to_anymap(settings))
+
+    property forward_rate_constants_temperature_derivatives:
+        """
+        Calculate Jacobian for forward rate constants with respect to temperature
+        at constant pressure, molar density and mole fractions.
+        """
+        def __get__(self):
+            return get_mapped(self, kin_fwdRateConstants_ddT, self.n_reactions)
+
+    property forward_rate_constants_pressure_derivatives:
+        """
+        Calculate Jacobian for forward rate constants with respect to pressure
+        at constant temperature, molar density and mole fractions.
+        """
+        def __get__(self):
+            return get_mapped(self, kin_fwdRateConstants_ddP, self.n_reactions)
+
+    property forward_rate_constants_density_derivatives:
+        """
+        Calculate Jacobian for forward rate constants with respect to molar density
+        at constant temperature, pressure and mole fractions.
+        """
+        def __get__(self):
+            return get_mapped(self, kin_fwdRateConstants_ddC, self.n_reactions)
+
+    property forward_rop_temperature_derivatives:
```

Comment:
I'm pretty torn on this: `forward_rop_temperature_derivatives` (36 characters) would become `forward_rates_of_progress_temperature_derivatives` (50 characters).

Reply:
Well, `forward_rate_constants_temperature_derivatives` (which already exists) is 46 characters, so 4 more isn't all that much.

But now that you mention it, abbreviating `T`, `P`, and `X` is actually consistent with the Python interface, where those abbreviations are already used. So perhaps either the `_ddT` suffix like in C++ would be reasonable, or something like `forward_rates_of_progress_T_derivatives` which is "only" 39 characters. I'm less sure about `C` as an abbreviation for the Python module.

Reply:
ok. Since `_ddT` etc are already used in the C++ API, and uppercase letters are valid in some instances, I am going with the shortest version after all.